extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -37,21 +38,97 @@ require([
 					.html(mustache.render(templates.ShareModal, {
 						heading: $.msg('embeddable-discussions-share-heading'),
 						icons: sharing.getData(mw.config.get('wgUserLanguage'), link, title),
+						close: $.msg('embeddable-discussions-cancel-button'),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this trailing comma. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ATrailingComma)